idrop-swing/src/main/resources/idrop.properties

```diff
@@ -1,4 +1,4 @@
-idrop.version=2.0.1-RC1
+idrop.version=4.3.7.0-SNAPSHOT
```

Comment:
What's the motivation behind this being 4.3.7.0-SNAPSHOT instead of something like 3.0.0?

Is bumping to 4.3.7.0-SNAPSHOT a requirement?

Reply:
Hmm, the 2.0.x version didn't align with the README... which makes your change feel more correct.